README.md

```diff
@@ -35,6 +35,19 @@ SUBCOMMANDS:
     help                 Prints this message or the help of the given subcommand(s)
 ```
 
+## Contract build config
+
+Building the contract uses `cargo-xbuild` under the hood for optimum Wasm binary size. This requires the following
+configuration section to be added to your contract's `Cargo.toml`:
```

Comment:
Cannot this be automated with the temporary `Cargo.toml` file?

Reply:
Yes, but it should be there permanently in the config.